server/channels/api4/team.go

```diff
@@ -1685,10 +1685,21 @@ func setTeamIcon(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	patchedTeam, err := c.App.GetTeam(c.Params.TeamId)
+	if err != nil {
+		c.Err = err
+		return
+	}
+
+	c.App.SanitizeTeam(*c.AppContext.Session(), patchedTeam)
+
 	auditRec.Success()
+	auditRec.AddEventResultState(patchedTeam)
 	c.LogAudit("")
 
-	ReturnStatusOK(w)
+	if err := json.NewEncoder(w).Encode(patchedTeam); err != nil {
+		c.Logger.Warn("Error while writing response", mlog.Err(err))
+	}
```

Comment:
In the description, it's said:

> This was due to after a patching a team, the updated team is sent to everyone via websockets. However, this team has been sanitized and therefore `invite_id` has been cleared

Should we fix the sanitization for the websocket event instead? Should `invite_id` be present in the WS event?

Reply:
No, it is intentionally being sanitized and should not be sent to all users.